.claude/commands/automation-publish.md

```
@@ -67,6 +70,18 @@ The `jleechanorg-pr-automation` package depends on `jleechanorg-orchestration`.
    print('SUCCESS: Both packages installed from PyPI')
    "
    ```
+4. **CRITICAL VERIFICATION** - Verify CLI validation imports resolve from the installed packages (still from /tmp):
+   ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)